Description: Fix compilation with jdk11
Author: Mathieu Malaterre <malat@debian.org>
Forwarded: no
Last-Update: 2020-02-17

--- fop-2.4.orig/fop-core/src/test/java/org/apache/fop/pdf/TableHeaderScopeTestCase.java
+++ fop-2.4/fop-core/src/test/java/org/apache/fop/pdf/TableHeaderScopeTestCase.java
@@ -112,7 +112,7 @@ public class TableHeaderScopeTestCase {
         return argThat(new IsScopeAttribute(scope));
     }
 
-    private static class IsScopeAttribute extends ArgumentMatcher<PDFDictionary> {
+    private static class IsScopeAttribute implements ArgumentMatcher<PDFDictionary> {
 
         private final Scope expectedScope;
 
@@ -121,7 +121,7 @@ public class TableHeaderScopeTestCase {
         }
 
         @Override
-        public boolean matches(Object argument) {
+        public boolean matches(PDFDictionary argument) {
             PDFDictionary attribute = (PDFDictionary) argument;
             return "/Table".equals(attribute.get("O").toString())
                     && expectedScope.getName().toString().equals(attribute.get("Scope").toString());
--- fop-2.4.orig/fop-core/src/test/java/org/apache/fop/render/intermediate/IFStructureTreeBuilderTestCase.java
+++ fop-2.4/fop-core/src/test/java/org/apache/fop/render/intermediate/IFStructureTreeBuilderTestCase.java
@@ -143,7 +143,7 @@ public class IFStructureTreeBuilderTestC
         return atts;
     }
 
-    private static final class AttributesMatcher extends ArgumentMatcher<Attributes> {
+    private static final class AttributesMatcher implements ArgumentMatcher<Attributes> {
 
         private final Attributes expected;
 
@@ -155,7 +155,7 @@ public class IFStructureTreeBuilderTestC
             return argThat(new AttributesMatcher(expected));
         }
 
-        public boolean matches(Object attributes) {
+        public boolean matches(Attributes attributes) {
             return attributesEqual(expected, (Attributes) attributes);
         }
 
